src/middleware/authentication/middleware.ts

```diff
@@ -24,18 +26,29 @@ export async function authenticationMiddleware(
     return response.sendStatus(403);
   }
 
-  const formId = request.params.formId;
-
   if (introspectionResult.username !== formId) {
+    logEvent(
+      introspectionResult.username,
+      { type: "Form", id: formId },
```

Comment:
```suggestion
      { type: "ServiceAccount", id: formId },
```

Reply:
The `type` and `id` must always match as the subject.  Changing `Form` to `ServiceAccount` would also mean the `id` would need to change to `serviceAccountId`.

My reasoning to go with `Form` was the following:
The user with JWT ${introspection.username} attempted to access resources of type ${Form} with ${id} and is Access Denied.

Reply:
Oh ok I thought it was a mistake because you had an unused `ServiceAccount` option under the `AuditSubjectType` enum.

Reply:
All good.  I added the `ServiceAccount` for possible future use if ever we allow refreshing of the keys through the API Server Directly.